src/hmac/index.js

```diff
@@ -2,21 +2,16 @@
 
 const crypto = require('crypto')
 const lengths = require('./lengths')
-const nextTick = require('async/nextTick')
 
-exports.create = function (hash, secret, callback) {
+exports.create = async function (hash, secret) { // eslint-disable-line require-await
   const res = {
-    digest (data, cb) {
+    async digest (data) { // eslint-disable-line require-await
       const hmac = crypto.createHmac(hash.toLowerCase(), secret)
```

Comment:
Why do we lowercase the string here?

Reply:
Nevermind I see :)

Reply:
From what I can see, our docs have things like `SHA512` as values for this field but the `crypto` module expects it to be `sha512`.